### ISSUE TITLE
Unable to install modules on disk

### ISSUE THREAD

**Issue by micka-fdz** [PrestaShop/PrestaShop#34642]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

Modules on disk do not appear in the backoffice, so it's impossible to install them. Works great in v8.

### Expected behavior

Modules on disk must appear in the backoffice to be installed.

### Steps to reproduce

1. Connect to backoffice
2. Try to install module on disk
3. Module do not appear

### PrestaShop version(s) where the bug happened

1.7.8.x

### Your company or customer's name goes here (if applicable).

@Creabilis

**Comment by micka-fdz** [comment]
The problem may be linked to the `ps_mbo` module, but I'm not sure.

Ping @sowbiba 🤔

**Comment by Hlavtox** [comment]
Could be related to module list caching.

Or another issue - if ps_distributionapi is installed, it's impossible to use a local version of this module, it gets replaced by the api version.

**Comment by micka-fdz** [comment]
@Hlavtox problem also appears on custom modules that are not on Addons...
We need to keep possibility to install them with "Install" button.

**Comment by Hlavtox** [comment]
@micka-fdz Aaaah you are using 1.7.8. It never worked dude, it was fixed in 8.0. :-))

In pre 8.0, you need to go to module catalog and there you will find the modules on disk, that are not installed.

**Comment by micka-fdz** [comment]
I know that but "Modules catalog" tab is not present anymore in 1.7.8. Issue is here.
The only solution is to install module with CLI, but this is not acceptable for a merchant.

**Comment by micka-fdz** [comment]
After a quick search, I think issue is related to https://github.com/PrestaShopCorp/ps_mbo/issues/612

**Comment by hibatallahAouadni** [comment]
Hello @micka-fdz 

Did you check @intraordinaire reply on the ps_mbo issue? Can I clsoe this issue?
Please check and feedback.

Thanks!

**Comment by intraordinaire** [comment]
@hibatallahAouadni, this issue is indeed more in the MBO scope and can be closed here IMO.
Feel free to refer to the [MBO issue ](https://github.com/PrestaShopCorp/ps_mbo/issues/612) mentioned before if you have other similar issues in the future.

**Comment by micka-fdz** [comment]
@intraordinaire I not agree, this issue is a regression in PS 1.7.8.x versions because of recent updates on `ps_mbo` module and solution can't be "Upgrade to PS 8.x.x" or "Upload module ZIP" ...

**Comment by paulnoelcholot** [comment]
Hello @micka-fdz,

I'm sorry but version 1.7.8 is no longer maintained. I invite you to upgrade your shop to 8.1

Thank you

**Comment by jf-viguier** [comment]
@paulnoelcholot 1.7.8 is in Extended Support Phase: cf 
https://build.prestashop-project.org/news/2023/178-in-extended-support-phase/

This issue has a Topwatchers tag so it seems a critical bug that can be fixed in this phase.
Please reopen the issue.

**Comment by intraordinaire** [comment]
@jf-viguier 
It is indeed in Extented Support, but this bug is not directly linked to the core, so this issue can be closed.
The regression is due to the evolution of the `ps_mbo` module.
I've reopened the issue in the [ps_mbo repo ](https://github.com/PrestaShopCorp/ps_mbo/issues/612), as we maybe find a simple way to re add this functionality.
In the meantime, please consider using workarounds specified is the related issue, as it's quite easy to implement, and it will unblock you faster.

**Comment by Progi1984** [comment]
@intraordinaire In 1.7, `ps_mbo` was in the Core. It left in 8.0 version.

**Comment by paulnoelcholot** [comment]
Hello @micka-fdz, indeed version 1.7 is in extended support. However, this only concerns critical bugs and security vulnerabilities. For a bug to be considered critical, the bug must stop the operation of the shop, which is not the case with this issue. This is therefore a major bug and is not therefore taken into account by extended support.

**Comment by micka-fdz** [comment]
Thanks everyone for your answers! And thanks @intraordinaire for reopening issue on `ps_mbo` repo 🙏
